Source/LibationSearchEngine/SearchEngine.cs

```diff
@@ -41,7 +41,7 @@ private static bool isAuthorNarrated(Book book)
             { FieldType.String, lb => lb.Book.Publisher, nameof(Book.Publisher) },
             { FieldType.String, lb => lb.Book.SeriesNames(), "SeriesNames", "Narrator", "Series" },
             { FieldType.String, lb => string.Join(", ", lb.Book.SeriesLink.Select(s => s.Series.AudibleSeriesId)), "SeriesId" },
-            { FieldType.String, lb => lb.Book.CategoriesIds() is null ? null : string.Join(", ", lb.Book.CategoriesIds()), nameof(Book.Category), "Categories", "CategoriesId", "CategoryId", "CategoriesNames" },
+            { FieldType.String, lb => lb.Book.CategoriesIds() is null ? null : string.Join(", ", lb.Book.CategoriesIds()), "Category", "Categories", "CategoriesId", "CategoryId", "CategoriesNames" },
```

Comment:
Also, I think this is a bug. All "category" fields relate only to the category ID, and none reference the category names.

It's been this way since before my refactor:
https://github.com/rmcrackan/Libation/blob/e62f01d2a39e02dd77d8f54fb15742878666e282/Source/LibationSearchEngine/SearchEngine.cs#L65-L69